<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notepad</class>
 <widget class="QMainWindow" name="Notepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notepad</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionGuardar_como"/>
    <addaction name="separator"/>
    <addaction name="actionBorrar"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionCopiar"/>
    <addaction name="actionCortar"/>
    <addaction name="actionPegar"/>
    <addaction name="actionElegir_todo"/>
    <addaction name="actionElegir_nada"/>
   </widget>
   <widget class="QMenu" name="menuVista">
    <property name="title">
     <string>Vista</string>
    </property>
    <addaction name="actionToolBarra_arriba"/>
    <addaction name="actionToolBarra_abajo"/>
    <addaction name="actionToolBarra_izquierda"/>
    <addaction name="actionToolBarra_derecha"/>
    <addaction name="separator"/>
    <addaction name="actionToolBarra_flotable"/>
    <addaction name="actionTool_Barra_movible"/>
   </widget>
   <widget class="QMenu" name="menuInsertar">
    <property name="title">
     <string>Insertar</string>
    </property>
    <addaction name="actionProductos"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEditar"/>
   <addaction name="menuVista"/>
   <addaction name="menuInsertar"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNuevo"/>
   <addaction name="actionGuardar"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionAbrir"/>
   <addaction name="actionBorrar"/>
  </widget>
  <action name="actionNuevo">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/iconos/simbolos2/file.png</normaloff>:/iconos/simbolos2/file.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/iconos/simbolos2/folder.png</normaloff>:/iconos/simbolos2/folder.png</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/iconos/simbolos2/disk.png</normaloff>:/iconos/simbolos2/disk.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionGuardar_como">
   <property name="text">
    <string>Guardar como</string>
   </property>
  </action>
  <action name="actionBorrar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/iconos/simbolos1/application_delete.png</normaloff>:/iconos/simbolos1/application_delete.png</iconset>
   </property>
   <property name="text">
    <string>Borrar</string>
   </property>
   <property name="toolTip">
    <string>Borrar</string>
   </property>
  </action>
  <action name="actionCopiar">
   <property name="text">
    <string>Copiar</string>
   </property>
  </action>
  <action name="actionCortar">
   <property name="text">
    <string>Cortar</string>
   </property>
  </action>
  <action name="actionPegar">
   <property name="text">
    <string>Pegar</string>
   </property>
  </action>
  <action name="actionElegir_todo">
   <property name="text">
    <string>Elegir todo</string>
   </property>
  </action>
  <action name="actionElegir_nada">
   <property name="text">
    <string>Elegir nada</string>
   </property>
  </action>
  <action name="actionToolBarra_arriba">
   <property name="text">
    <string>ToolBarra arriba</string>
   </property>
  </action>
  <action name="actionToolBarra_abajo">
   <property name="text">
    <string>ToolBarra abajo</string>
   </property>
  </action>
  <action name="actionToolBarra_izquierda">
   <property name="text">
    <string>ToolBarra izquierda</string>
   </property>
  </action>
  <action name="actionToolBarra_derecha">
   <property name="text">
    <string>ToolBarra derecha</string>
   </property>
  </action>
  <action name="actionToolBarra_flotable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ToolBarra flotable</string>
   </property>
  </action>
  <action name="actionTool_Barra_movible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tool Barra movible</string>
   </property>
  </action>
  <action name="actionProductos">
   <property name="text">
    <string>Productos</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
